posthog-rails/examples/posthog.rb

```diff
@@ -0,0 +1,151 @@
+  config.auto_capture_exceptions = true
+
+  # Report exceptions that Rails rescues (e.g., with rescue_from) (default: true)
+  config.report_rescued_exceptions = true
+
+  # Automatically instrument ActiveJob background jobs (default: true)
+  config.auto_instrument_active_job = true
+
+  # Capture user context with exceptions (default: true)
+  config.capture_user_context = true
+
+  # Controller method name to get current user (default: :current_user)
+  # Change this if your app uses a different method name
+  config.current_user_method = :current_user
+
+  # Additional exception classes to exclude from reporting
+  # These are added to the default excluded exceptions
+  config.excluded_exceptions = [
+    # 'MyCustom404Error',
+    # 'MyCustomValidationError'
+  ]
+
+  # ============================================================================
+  # ENVIRONMENT-SPECIFIC CONFIGURATION
+  # ============================================================================
+
+  # Disable in test environment
+  config.test_mode = true if Rails.env.test?
+
+  # Optional: Disable in development
+  # if Rails.env.development?
+  #   config.test_mode = true
+  # end
+end
+
+# ============================================================================
+# DEFAULT EXCLUDED EXCEPTIONS
+# ============================================================================
+# The following exceptions are excluded by default:
+#
+# - AbstractController::ActionNotFound
+# - ActionController::BadRequest
+# - ActionController::InvalidAuthenticityToken
+# - ActionController::InvalidCrossOriginRequest
+# - ActionController::MethodNotAllowed
+# - ActionController::NotImplemented
+# - ActionController::ParameterMissing
+# - ActionController::RoutingError
+# - ActionController::UnknownFormat
+# - ActionController::UnknownHttpMethod
+# - ActionDispatch::Http::Parameters::ParseError
+# - ActiveRecord::RecordNotFound
+# - ActiveRecord::RecordNotUnique
+#
+# These can be re-enabled by removing them from the exclusion list if needed.
+
+# ============================================================================
+# USAGE EXAMPLES
+# ============================================================================
+
+# Track custom events:
+# PostHog.capture(
+#   distinct_id: current_user.id,
+#   event: 'user_signed_up',
+#   properties: {
+#     plan: 'pro',
+#     source: 'organic'
+#   }
+# )
+
+# Identify users:
+# PostHog.identify(
+#   distinct_id: current_user.id,
+#   properties: {
+#     email: current_user.email,
+#     name: current_user.name,
+#     plan: current_user.plan
+#   }
+# )
+
+# Check feature flags:
+# if PostHog.is_feature_enabled('new-checkout-flow', current_user.id)
+#   render 'checkout/new'
+# else
+#   render 'checkout/old'
+# end
+
+# Capture exceptions manually:
+# begin
+#   dangerous_operation
+# rescue => e
+#   PostHog.capture_exception(e, current_user.id, { context: 'manual' })
+#   raise
+# end
```

Comment:
I don't think this makes sense here. This file should be kept as a reasonable default initializer for our customers.

Also, can you add a new `bundle posthog:install` command that automatically copies this file over to `config/initializers/posthog.rb` and document that?